include/ostatus.php

```diff
@@ -1981,7 +2001,7 @@ public static function feed(&$a, $owner_nick, $last_update) {
 				dbesc($owner["nurl"]), dbesc(str_replace("http://", "https://", $owner["nurl"])),
 				dbesc($owner["nurl"]), dbesc(str_replace("http://", "https://", $owner["nurl"]))
 			);
-
+*/
```

Comment:
I added a date and a comment. If the new query has proven to be a good replacement, we can remove the old one. (I suggest when changing from "develop" to "master")